propertyestimator/workflow/groups.py

```diff
@@ -749,6 +753,12 @@ def _evaluate_condition(self, condition_type, left_hand_value, right_hand_value)
         if left_hand_value is None or right_hand_value is None:
             return False
 
+        if isinstance(left_hand_value, EstimatedQuantity):
+            left_hand_value = left_hand_value.uncertainty
+            
+        if isinstance(right_hand_value, EstimatedQuantity):
+            right_hand_value = right_hand_value.uncertainty
```

Comment:
Should the condition be explicit about whether the user want to specify an uncertainty or an estimated quantity instead? For example, what if I want to check something like `EstimatedQuantity(1.2*kcal/mol, uncertainty=2.0*kcal/mol) < 1.5*kcal/mol`, which, if we're interested in the statistical significance, would return `False`?

Reply:
So theres two parts to this and neither are particularly satisfactory unfortunately..

At the moment we have to hard code in that if we have passed in an estimated quantity we only look at the uncertainty because the ProtocolPath class is not generic enough at the moment to be able to point to properties within classes - but this is coming in the future.

The other thing is for now I've disabled <, > checks on EstimatedQuantity until I can think them through, because I'm not 100% sure what the expected behaviour of them is (and same with ==) should we be comparing whether one thing is significantly (in a statistical sense) less than another thing, or should we check whether one thing is less than the other thing to within machine precision.

Reply:
> I've disabled <, > checks on EstimatedQuantity until I can think them through, because I'm not 100% sure what the expected behaviour of them is

I see. That probably deserves its own discussion/PR.